internal/pkg/client/cache/dir.go

```diff
@@ -85,13 +230,14 @@ func initCacheDir(dir string) error {
 
 // cleanAllCaches is an utility function that wipes all files in the
 // cache directory, will return a error if one occurs
-func cleanAllCaches() {
+func (c *ImgCache) cleanAllCaches() {
 	// TODO: add oras here
```

Comment:
this TODO is done, no?

Reply:
No it is not finalized and outside out-the-scope of this PR (testing for example is missing, including testing of ORAS clean)